agent/src/agent/data_extractor/observium/observium.py

```diff
@@ -31,18 +31,21 @@ def _get(pipeline_: Pipeline):
 def _create_metrics(data: dict, pipeline_: Pipeline) -> list:
     metrics = []
     # these values must be outside the for loop for optimization purposes
-    fields = field.build_fields(pipeline_.dimension_configurations)
-    value_paths = pipeline_.value_paths
-    value_paths = dict(zip(value_paths, value_paths))
+    fields_dims = field.build_fields(pipeline_.dimension_configurations)
+    fields_meas = field.build_fields(pipeline_.values_configurations)
+    fields_tags = field.build_fields(pipeline_.tags_configurations)
     schema_id = pipeline_.get_schema_id()
     try:
         for obj in data:
             metric = {
                 "timestamp": obj[pipeline_.timestamp_name],
-                "dimensions": field.extract_fields(fields, obj),
-                "measurements": _extract_measurements(obj, value_paths),
+                "dimensions": field.extract_fields(fields_dims, obj),
+                "measurements": field.extract_fields(fields_meas, obj),
+                "tags":  pipeline_.tags | {name: [tags] for name, tags in field.extract_fields(fields_tags, obj).items()},
```

Comment:
`pipeline_.tags` will be added in streamsets so it's not needed here

Reply:
Agree. Fixed